src/lib.rs

```diff
@@ -1,14 +1,28 @@
 #[cfg(feature = "gaiku_baker_heightmap")]
+pub use gaiku_baker_heightmap as baker_heightmap;
+#[cfg(feature = "gaiku_baker_heightmap")]
 pub use gaiku_baker_heightmap::*;
```

Comment:
I don't see the point on exposing in two different ways, we should stick with one, I picked to just re-export everything from the sub-crates because in most cases they have just one public struct.

Reply:
So you think just leave it as `::*`? Ok

Reply:
Ok removed these rexports